confirmation/models.py

```diff
@@ -100,6 +100,16 @@ def get_link_validity_in_days(self):
         # type: () -> int
         return settings.EMAIL_CHANGE_CONFIRMATION_DAYS
 
+class RemoteServerRegistrationManager(ConfirmationManager):
+    def get_activation_url(self, key, host=None):
```

Comment:
They are different. We override this function to map to `zilencer.views.confirm`. In `ConfirmationManger`, this function maps to `confirmation.views.confirm`.